doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -758,42 +801,205 @@ Response:
             "value": 21.7
         }
 
-### Get a single attribute value. Not exists [GET /v2/entities/{entityId}/attrs/{attrId}]
+### Update attribute data [PUT /v2/entities/type/{entityType}/id/{entityId}/attrs/{attrName}]
+
+The request payload is an object representing the new attributes data. The attributes data previously existing
+is removed and replaced by the one in the request. The object follows the JSON representation for attributes
+(described in a section above).
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
 
 + Parameters
     + entityId: Bcn_Welt (required, string) - Entity ID
-    + attrId: mistake (required, string) - Attribute to be retrieved.
+    + entityType: Room (required, string) - Entity Type
+    + attrName: temperature (required, string) - Attribute ID
 
-+ Response 404 (application/json)
++ Request (application/json)
 
         {
-            "error": "NotFound",
-            "description": "The entity does not have such an attribute"
+            "value": 25.0,
+            "unitCode": "CEL"
         }
 
-### Update an attribute value - JSON [PUT /v2/entities/{entityId}/attrs/{attrId}]
++ Response 200
+
+
+### Remove a single attribute [DELETE /v2/entities/type/{entityType}/id/{entityId}/attrs/{attrName}]
+
+Removes the attribute from the entity.
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + entityType: Room (required, string) - Entity Type
+    + attrName: temperature (required, string) - Attribute ID
+
++ Response 204
+
+## Entity Attribute Value [/v2/entities/{entityId}/attrs/{attrName}/value]
+
+### Get attribute data [GET /v2/entities/{entityId}/attrs/{attrName}/value{?text}]
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">options</td>
+  <td>Operation opations. Allowed values: <code>text</code>.</td>
+</tr>
+</table>
+
+If the `text` option is not used, returns a JSON object which the `value` of the attribute. If the
```

Comment:
Even although JSON is the default, it should be explained the structure of that JSON. However, I will try to explain it in a different way, taking into account your suggestion.